report/2.Analysis/LanguagesForYoungPeople.tex

```diff
@@ -0,0 +1,63 @@
+\section{Languages for beginners}
+
+There are a wide variety of different tools, games and languages with the desired intend of learning students how to program. 
+These span from very specific game creation tools, to general purpose programming languages. 
+In order to gather some inspiration and knowledge about existing solutions, some of the most common languages for beginners were analysed.
```

Comment:
* State what we as authors are gathering inspiration for, by analyzing the languages below.